util/unifyfs/src/unifyfs-rm.c

```diff
@@ -271,6 +368,27 @@ static int remove_server_pid_file(unifyfs_args_t* args)
     return ret;
 }
 
+static int remove_stage_status_file(unifyfs_args_t* args)
+{
+    int ret = 0;
+    char filename[PATH_MAX] = { 0, };
+
+    sprintf(filename, "%s/%s", args->share_dir, UNIFYFS_STAGE_STATUS_FILENAME);
```

Comment:
use `scnprintf()`

Reply:
Done for all occurrences of sprintf() in this file.  

I will point out that sprintf appears at least 126 times elsewhere in the unifyfs source code.